src/Views/batchprocessor.html

```diff
@@ -104,16 +136,53 @@ <h3>Select Basin Characteristics</h3>
                         </tr>
                     </tbody>
                 </table>
+            </div><!-- End parameter selection -->
 
-            </div>
-            <!-- End submit batch -->
+            <!-- Start user information -->
+            <div style="padding-top: 15px;" class="row">
+                <div class="col-md-5"> ID Field: 
+                    <i ng-mouseover="showTooltip = true" ng-mouseleave="showTooltip = false" class="fa fa-question-circle"></i>
+                    <span ng-show="showTooltip" class="report-tooltip">
+                        <span>This is the field heading from the shapefile of points that is unique to each point and allows the
+                            data to be
+                            joined with the batch results.</span>
+                    </span>
+                    <input type="text" id="idField" ng-model="vm.submitBatchData.idField" required>
+                </div>
+                <div class="col-md-7">
+                    <span> Email Address: 
+                        <input type="text" id="emailField" ng-model="vm.submitBatchData.email" required></span>
+                </div>
+            </div> <!-- End user information-->
 
-            <!-- Start batch status -->
-            <div id="batchStatus" ng-if="vm.selectedBatchProcessorTabName==='batchStatus'">
-                <p>This is the batch status tab</p>
+            <!-- Create a file upload button and specify it needs to be a zipped shapefile -->
+            <div style="padding-top: 15px;">
+                <span>Upload Shapefile (.zip):</span><br />
+                <label for="shapefile" style="border: 1px solid #ccc; display: inline-block; padding: 6px 12px; cursor: pointer;"> Select file</label>
+                <input type="file" ngf-select ng-model="vm.submitBatchData.attachment" id="shapefile" name="shapefile" accept=".zip" ngf-accept="application/zip" required style="display: none;">
+                <span ng-show="vm.submitBatchData.attachment">
+                    <i class="fa fa-check" style="color:green"></i>&nbsp;&nbsp;{{vm.submitBatchData.attachment.name}}<button class="close removeItem"
+                        style="" ng-click="vm.submitBatchData.attachment = null">&times;</button>
+                </span>
             </div>
-            <!-- End batch status -->
-        </div>
+
+            <!-- Button to submit batch  -->
+            <div style="padding-top: 15px; text-align: center;">
+                <button type="submit" class="btn-wim-default" ng-click="vm.submitBatch()"
+                    ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email"><i class="fa fa-share"></i><span>&nbsp;&nbsp;Submit Batch</span></button>

```

Comment:
I think this will prevent batch submission until user has uploaded a file

```suggestion
                    ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email || !vm.submitBatchData.attachment"><i class="fa fa-share"></i><span>&nbsp;&nbsp;Submit Batch</span></button>

```

Reply:
I previously tried this and wasn't working for me but I will test it again.